debezium-quarkus-outbox/integration-tests/pom.xml

```diff
@@ -35,6 +35,13 @@
                 <version>${project.version}</version>
                 <scope>test</scope>
             </dependency>
+
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-testing-testcontainers</artifactId>
+                <version>${project.version}</version>
```

Comment:
The dependency version should imo be moved to the parent pom, shouldn't it @jpechane ?

Reply:
Yes, this should go into `dependencyManagement` of the parent POM.